src/esp/gfx/GenericDrawable.h

```diff
@@ -49,6 +50,8 @@ class GenericDrawable : public Drawable {
   Magnum::Resource<LightSetup> lightSetup_;
 
   Magnum::Shaders::PhongGL::Flags flags_;
+
+  bool isRendererCreated;
```

Comment:
Here is what I think:
When renderer is NOT created, we do not need to create drawables. In other words, we will create them when the renderer is created as usual. Then why shall we change all the drawables? Is it because this is a temporary solution, and it is the easiest way to hack it in such a manner?

Reply:
We're creating a Drawable because this flag is meant to be minimally-invasive. The fewer lines of code this PR changes, the easier it will be to ensure this flag's correctness and maintain it.

If we disabled creating a Drawable, we'd have to make lots of other changes in the code, for example, places where we call `Drawable::setLightSetup`.

This flag is not meant to aggressively avoid loading as much data as possible for the case of no rendering. We expect that will be a separate flag, something like "loadRenderAssets".

Reply:
Make sense. You explanation aligned with my guess. Thank you.